src/tests/utils/convert.js

```diff
@@ -94,173 +105,176 @@ describe('convert render functions', () => {
   });
 
   describe('authentication', () => {
-    it('should render basic auth', (done) => {
+    it('should render basic auth', () => {
       const wbDef = definitions.basic;
 
-      convert.renderAuth(wbDef)
+      return convert.renderAuth(wbDef)
```

Comment:
To make the code cleaner, instead of using `done()` callback, I changed the style of all the auth-related tests.